examples/hls_sink.exs

```diff
@@ -86,6 +86,11 @@ defmodule Example do
   end
 end
 
+# stop executing when running on CircleCI
+if System.get_env("CIRCLE_CI") do
+  System.halt(0)
+end
+
```

Comment:
Do we need special behaviour for CI in this case?

Reply:
I think so, we don't need to create h264 file and wait for the pipeline to finish playing, right?

Reply:
why not? :P instead of creating h264 pipeline we can get it via hackney source from [https://github.com/membraneframework/static/blob/gh-pages/video-samples/test-video.h264]

Reply:
Works like a charm :)